src/OctoshiftCLI.Tests/GithubApiTests.cs

```diff
@@ -1665,7 +1665,7 @@ public async Task RepoExists_Should_Return_False_If_Repo_Does_Not_Exist_301()
         }
 
         [Fact]
-        public async Task RepoExists_Throws_When_Underlying_HtttRepsponseException_Status_Is_Not_NotFound_Or_Moved()
+        public async Task RepoExists_Throws_When_Underlying_HtttResponseException_Status_Is_Not_NotFound_Or_Moved()
```

Comment:
I'm assuming that these functions don't care about their names.
```diff
-HtttRepsponse
+HtttResponse
```

(I also suspect that `Httt` should be `Http`...)